catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.querymonitor.xml

```diff
@@ -23,6 +23,11 @@
             name="Query Timeout" id="queryTimeoutMinutes" required="true" type="Long"
             default="5"/>
 
+        <AD description="Set the interval in minutes for the query results in the workspace (Range: 0 - 1440 (24 hours)).
+                         Note:  This will query for results from the interval to the time the query is sent out."
+            name="Notification Time Interval" id="queryTimeInterval" required="true" type="Integer"
+            default="1440"/>
+
         <AD description="Set the cron string trigger."
             name="Cron String" id="cronString" required="true" type="String"
```

Comment:
I realize this is outside of the scope of this PR, but these names poorly reflect what the configuration is being used for. Recommend renaming `Cron String` to be `Email Subscription Interval (Cron Expression)`. As well as adding even a single cron expression example in the help description.